Services/Converters/ArtefactCommentConverter.cs

```diff
@@ -0,0 +1,44 @@
+using System.Collections.Generic;
+using System.Linq;
+
+using Artefactor.Models;
+
+namespace Artefactor.Services.Converters
+{
+    public class ArtefactCommentConverter : IConverter<ArtefactComment>
+    {
+        public object ToJson(ArtefactComment c)
```

Comment:
The returned objects should have some kind of `type: [question|comment]` field, because ATM I don't think there's any way for the frontend to distinguish, other than hacky undef checks of `AnswerCommentId`/`IsAnswered`?

Reply:
I definitely overlooked that. Will push shortly.